packages/@aws-cdk/aws-scheduler-alpha/test/schedule-expression.test.ts

```diff
@@ -94,7 +94,7 @@ describe('schedule expression', () => {
   });
 
   test('one-time expression string has expected date', () => {
-    const x = ScheduleExpression.at(new Date(2022, 10, 20, 19, 20, 23));
+    const x = ScheduleExpression.at(new Date(Date.UTC(2022, 10, 20, 19, 20, 23)));
```

Comment:
Thanks! Good catch!

Reply:
Yeah, we discussed it internally in Slack (pinged you again) and @Jacco suggestion was: 

```
public static at(year: int, monthIndex: int, day?: int, hours?: int, minutes?: int, seconds?: int, milliseconds?: int, timeZone?: TimeZone): ScheduleExpression {
   ...
}
```

Reply:
I still think that `at` should just mirror what we have elsewhere. If we want to change how `at` works cdk-wide fine, but thats a different conversation